test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -929,9 +653,7 @@ def test_mask_rcnn(self):
     def test_heatmaps_to_keypoints(self):
         maps = torch.rand(10, 1, 26, 26)
         rois = torch.rand(10, 4)
-        from torchvision.models.detection.roi_heads import (
-            heatmaps_to_keypoints,
-        )
+        from torchvision.models.detection.roi_heads import heatmaps_to_keypoints
```

Comment:
I would move this out to the top to make the dependency clear.

Reply:
+1